src/ec/packages/ec-component-icon/icon.story.js

```diff
@@ -3,11 +3,11 @@ import { withKnobs, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
 
-import brandedIcons from '@ecl/ec-resources-icons/dist/lists/branded.json';
-import generalIcons from '@ecl/ec-resources-icons/dist/lists/general.json';
-import notificationsIcons from '@ecl/ec-resources-icons/dist/lists/notifications.json';
-import uiIcons from '@ecl/ec-resources-icons/dist/lists/ui.json';
-import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import brandedIcons from '@ecl/eu-resources-icons/dist/lists/branded.json';
```

Comment:
I guess there is a confusion between EC and EU here...
EC components (as this file), should use EC dependencies, no EU, and the same for the other way

Reply:
Fixed here https://github.com/ec-europa/ecl-twig/pull/15/commits/7b43a8964e4a47f6a5d930518de753c95058232d

That was stupid mistake